### PR TITLE
10 fix bugs truly unlimited speaker colors and audio distinction

### PR DESCRIPTION
**Pull request by EvaMach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,6 @@ const container = document.querySelector('#publish-container');
 
 const publish = new BeeyPublish(container, {
   media: {
-    url: '<url of audio/vidoe file>',
```
Comment (podlomar) [inline README.md:29]: Why did you remove this line? The media URL is still a valid property, isn't it?

**Review by podlomar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EvaMach) [reply README.md:29]: I must have done that by mistake. Sorry! I will fix it in the next commit.
